sources.list

```diff
@@ -0,0 +1,4 @@
+deb http://deb.debian.org/debian buster main contrib non-free
```

Comment:
Should we use "stable" instead of "buster" so hopefully we dont have to worry about it on the next release?

Reply:
Good point